components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -1025,6 +1028,8 @@ void HommeDynamics::initialize_homme_state () {
   const auto ps0 = hvcoord.ps0 * hvcoord.hybrid_ai0;
 
   const auto policy = ESU::get_thread_range_parallel_scan_team_policy(nelem*NGP*NGP,npacks_mid);
+  const auto phis = m_helper_fields.at("phis_dyn").get_view<const Real***>();
```

Comment:
Since there still is a `phis` and `phis_dyn`, should this be called `const auto phis_dyn =` to avoid confusion? Feel free to ignore.

Reply:
Yeah, let's prevent any future confusion.